evap/results/tests/test_views.py

```diff
@@ -772,120 +771,212 @@ def test_textanswer_visibility_for_manager(self):
         self.assertNotIn(".responsible_contributor_additional_orig_hidden.", page)
 
 
-class TestResultsTextanswerVisibility(WebTest):
+# NOTE antworten für die zu testende person in eine liste packen, weil die je nach filter immer die gleichen assertions haben(?)
+# test
+class TestResultsTextanswerVisibilityForStudent(WebTest):
```

Comment:
If possible, it would be nice (and I think it's feasible) to get back to one test class for the whole (django) view